src/Kuzzle.js

```diff
@@ -239,6 +247,7 @@ class Kuzzle extends KuzzleEventEmitter {
 
     this.protocol.addListener('tokenExpired', () => {
       this.jwt = undefined;
+      this.jwtExpiresAt = undefined;
```

Comment:
It's just question from a JS noob, what is the difference between:
```js
this.jwtExpiresAt = undefined;
```
and
```js
delete this.jwtExpiresAt;
```

Reply:
With `this.jwt = undefined` the function `hasOwnProperty('jwt')` will still return true but in this case it's change nothing